server/log_rpc_server_test.go

```diff
@@ -267,7 +269,59 @@ func TestQueueLeaves(t *testing.T) {
 	server := NewTrillianLogRPCServer(mockRegistry, fakeTimeSource)
 
 	if _, err := server.QueueLeaves(context.Background(), &queueRequest0); err != nil {
-		t.Fatalf("Failed to get leaf by index: %v", err)
+		t.Fatalf("Failed to queue leaf: %v", err)
+	}
+}
+
+func TestQueueLeavesDuplicateErrorMapped(t *testing.T) {
+	ctrl := gomock.NewController(t)
+	defer ctrl.Finish()
+
+	type errMapTest struct {
```

Comment:
(nit) no need to name the new type - can just do:
```go
var tests = [] struct {
  err   error
  want codes.Code
￼} {
  ...
}
```
(and I believe "want" is preferred to "expected")